### PR TITLE
DEV-3451 docs(removeDuplicate):add subject

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

File: subjects/removeduplicate/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import (
+	"fmt"
+
+	"piscine"
+)
+
+func main() {
+	fmt.Println(piscine.RemoveDuplicate([]int{1, 2, 3, 3, 4, 5, 6, 7, 8, 9, 10}))
+	fmt.Println(piscine.RemoveDuplicate([]int{1, 1, 2, 2, 3}))
```
Comment (tiagocollot) [inline subjects/removeduplicate/README.md:31]: This is only a suggestion, maybe we can add one output that matches this condition: "If the slice is empty, return the slice itself."

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/removeduplicate/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+```
+
+And its output :
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/removeduplicate/README.md]: fixed

File: subjects/removeduplicate/README.md (hunk 1 above)
Comment (jotapero) [reply subjects/removeduplicate/README.md:32]: fixed

File: subjects/removeduplicate/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+## removeduplicate
+
+### Instructions
+
+Write a function that takes a slice of integers as an argument and returns the slice where all duplicate integers have been removed.
+- If the slice is empty, return the slice itself.
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/removeduplicate/README.md:7]: fixed

**Review by tiagocollot — APPROVED** [review]
LGTM! ⚡

**Pull request merged**